src/strategies/Primitives/LimitSwapExactInput.ts

```diff
@@ -0,0 +1,115 @@
+import { TokenArgs, PrimitiveParamType, BigIntish } from '@brinkninja/types'
+import Token from '../Token'
+import InputTokenPrimitive from './InputTokenPrimitive'
+
+export type LimitSwapExactInputArgs = {
+  priceCurveAddress: string
```

Comment:
upgrade to `brinkninja/types@0.9.6` and use the `PriceCurveJSON` type to combine these into 1 arg, similar to how MarketSwapExactInput primitive uses `OracleJSON`